### ISSUE TITLE
Improvement: Config file as parameters.

### ISSUE THREAD

**Issue by tdaxel** [destil/MoodSync#4]
I love your little program.
I use it in my living room but i would love to have someparamters to adjust.

1:My light are to the left of my television, could be great to say what part of the screen it should focus on. maybe just have All, left top right bottom, and make it easy and just take 50% of the screen when you select.

2: refresh rate my nvidia shield has plenty of power, so i would maybe like it to update more often. (parameter)

3: max light output. i think (you currently output light at 50% brightness, could be awesome to let the user select it them self.

4: if possible make more analyses run at the same palettes/screen so right side analyse outputs to "group x in lifx" left site outputs to "group y in lifx" etc.

Love to hear your comments, 
Maybe I can help you out with some of it.

/Thomas

**Comment by destil** [comment]
Hi, it would be cool to have more options like that. But this app is not my priority, I'm just maintaining it but I'm not planning new features. I would gladly accept pull request though!

**Comment by tdaxel** [comment]
i will look at it...

What is the best dev environment for android apps (don't do apps normally,
but more desktop applications where we use eclipse)

/Thomas

On Fri, Oct 16, 2015 at 4:09 PM, David Vavra <notifications@github.com>
wrote:

> Hi, it would be cool to have more options like that. But this app is not
> my priority, I'm just maintaining it but I'm not planning new features. I
> would gladly accept pull request though!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/destil/MoodSync/issues/4#issuecomment-148727552>.
>



-- 

Med venlig hilsen / Yours Sincerely,

*Thomas Axel*
Software Developer - Build
Configit A/S
Kristianiagade 7, 2100
Copenhagen, Denmark
taxel@configit.com
<http://www.configit.com/>

**Comment by destil** [comment]
Android Studio

**Comment by tdaxel** [comment]
I got i downloaded this weekend.

And seems to get it up running.


Dont seems like the emulator can change the lamps. dont even find them..
Is that a emulator setting or?

Sorry for asking all this :D

my private mail is cc'ed then i can also respond in evenings :)


/Thomas

On Fri, Oct 16, 2015 at 4:49 PM, David Vavra <notifications@github.com>
wrote:

> Android Studio
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/destil/MoodSync/issues/4#issuecomment-148737255>.
>



-- 

Med venlig hilsen / Yours Sincerely,

*Thomas Axel*
Software Developer - Build
Configit A/S
Kristianiagade 7, 2100
Copenhagen, Denmark
taxel@configit.com
<http://www.configit.com/>

**Comment by destil** [comment]
The emulator would have to be on the same wifi, which is tricky. I suggest you test on a real device.

**Comment by tdaxel** [comment]
yeah... also what i wanted.. but I have Samsung Knox installed, wich apparently does so you cant enable USB debugging :(
https://www.samsungknox.com/en/faq/usb-debugging-disabled-default-knox-22

So right now iam coding in blindness (as you might also see in my Fork i mate yesterday :-) )
